jooq/src/main/kotlin/com/terraformation/backend/jooq/Config.kt

```diff
@@ -157,8 +160,10 @@ val ID_WRAPPERS =
                     "GbifTaxonId",
                     listOf("gbif_taxa\\.id", "gbif_.*\\.taxon_id", "gbif_.*\\..*_usage_id")),
                 IdWrapper("InternalTagId", listOf("internal_tags\\.id", ".*\\.internal_tag_id")),
+                IdWrapper("ManagedFacilityTypeId", listOf(".*\\.managed_facility_type_id")),
```

Comment:
This should be included in the column list for the enum type rather than defined as an ID wrapper.